conda/plugins/types.py

```diff
@@ -24,11 +24,25 @@ class CondaSubcommand(NamedTuple):
 
 class CondaVirtualPackage(NamedTuple):
     """
-    Conda subcommand entry.
+    A conda virtual package.
 
     :param name: Virtual package name (e.g., ``my_custom_os``).
     :param version: Virtual package version (e.g., ``1.2.3``).
     """
 
     name: str
     version: str | None
+
+
+class CondaSolver(NamedTuple):
+    """
+    A conda solver.
+
+    :param name: Subcommand name (e.g., ``conda my-subcommand-name``).
```

Comment:
Let's update this 😁 (looks like it was copied over from the CondaSubCommand type.